odl/solvers/advanced/proximal_operators.py

```diff
@@ -1079,6 +1080,91 @@ def _call(self, x, out):
     return ProximalCConjKL
 
 
+def proximal_cconj_kl_version_two(space, lam=1, g=None):
+    """Proximal operator factory of the convex conjugate of the KL divergence.
+
+    Function returning the proximal operator of the convex conjugate of the
+    functional F where F is the entropy-type Kullback-Leibler (KL) divergence
+
+        F(x) = sum_i (xln(pos(x)) - xln(g) + g - x)_i + ind_P(x)
```

Comment:
prefer moving indices inside and use spacing, i.e. 

`sum_i g_i x_i ln(pos(x_i)) - x_i ln(g_i) + g_i - x_i`